src/ResourceManager/Resources/Commands.Resources/help/New-AzureRmRoleAssignment.md

```diff
@@ -141,6 +141,15 @@ PS C:\> New-AzureRMRoleAssignment -ObjectId 5ac84765-1c8c-4994-94b2-629461bd191b
 
 Grant access to a group at a nested resource (subnet)
 
+### --------------------------  Example 5  --------------------------
+```
+PS C:\> $servicePrincipal = New-AzureRmADServicePrincipal -DisplayName "testServiceprincipal"
+PS C:\> New-AzureRmRoleAssignment -RoleDefinitionName "Reader" -ServicePrincipalName $servicePrincipal.ApplicationId
```

Comment:
@darshanhs90 I believe the user from the issue is suggesting that `ServicePrincipalName` be renamed to `ApplicationId`. If that is the case, then we rename the parameter to `ApplicationId` and add an alias to `ServicePrincipalName`. What do you think?

Reply:
sure sounds good

Reply:
will update the pr once this [https://github.com/Azure/azure-powershell/pull/5023 ] gets merged,since i would need to rerecord the tests once i change the parameter

Reply:
@darshanhs90 that PR has been merged